src/app/forms/david-cowans.jade

```diff
@@ -18,6 +18,8 @@
             .small-12.columns
               textarea(placeholder="Establish credibility by sharing the pedigree of the entrepreneurs, customers, or the investors", ng-model="ctrl.method.pedigree", required=true)
           .row
+            span.has-tip(data-tooltip, aria-haspopup="true", title= "Every day journalists spends hours finding which information people needs to know about Public Figures. We offer a publisher network to collect millions of questions and give to the journalist the popular ones with the best question recognition technology") Example
```

Comment:
on the aria-haspopup attribute, does it needs to be the string "true" or could it be just true?
And the same for the other views.